worlds/sm64ex/Rules.py

```diff
@@ -38,48 +38,59 @@ def set_rules(world, player: int, area_connections):
 
     #Special Rules for some Locations
     add_rule(world.get_location("Tower of the Wing Cap Switch", player), lambda state: state.has("Power Star", player, 10))
-    add_rule(world.get_location("Cavern of the Metal Cap Switch", player), lambda state: state.can_reach("Hazy Maze Cave",'Region',player))
+    add_rule(world.get_location("Cavern of the Metal Cap Switch", player), lambda state: state.can_reach("Hazy Maze Cave", 'Region', player))
     add_rule(world.get_location("Vanish Cap Under the Moat Switch", player), lambda state: state.can_reach("Basement", 'Region', player))
 
+    add_rule(world.get_location("BoB: Mario Wings to the Sky", player), lambda state: state.has("Cannon Unlock BoB", player))
```

Comment:
I was told on Discord you can reach this Star without the Cannon Unlock, is that not true?

Reply:
A couple of us spent a while trying to get it, because I was skeptical. I spent half an hour trying to get the highest secret without getting close, and other people concluded the same, so even if it is possible I think it's too hard even for non-strict.